core/trino-main/src/main/java/io/trino/operator/DirectExchangeClientStatus.java

```diff
@@ -46,7 +49,8 @@ public DirectExchangeClientStatus(
             @JsonProperty("spilledPages") int spilledPages,
             @JsonProperty("spilledBytes") long spilledBytes,
             @JsonProperty("noMoreLocations") boolean noMoreLocations,
-            @JsonProperty("pageBufferClientStatuses") List<PageBufferClientStatus> pageBufferClientStatuses)
+            @JsonProperty("pageBufferClientStatuses") List<PageBufferClientStatus> pageBufferClientStatuses,
+            @JsonProperty("requestsDurationInMilliseconds") TDigestHistogram requestsDuration)
```

Comment:
`requestsDurationInMilliseconds` -> `requestDuration`

Reply:
ok, done. Anyway, the person who reads the json has no idea what is the unit of request duration.